stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/ChatUI.kt

```diff
@@ -45,13 +46,36 @@ public object ChatUI {
         }
 
     private var markdownOverride: ChatMarkdown? = null
-    private val defaultMarkdown: ChatMarkdown by lazy { ChatMarkdownImpl(appContext) }
+    private val defaultMarkdown: ChatMarkdown by lazy {
+        ChatMarkdown { textView, message ->
+            textView.text = message
+        }
+    }
+
+    @Deprecated(message = "ChatUI.markdown is deprecated. Use ChatUI.messageTextTransformer",
```

Comment:
Let's add more information here - we extracted markdown support to another module. See docs for more reference